<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gausss</class>
 <widget class="QMainWindow" name="Gausss">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gausss</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X1:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Limpia">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>490</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X3:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Sale">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Tabla">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>571</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <row>
     <property name="text">
      <string>Ecuacion 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ecuacion 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ecuacion 3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Ecuacion 4</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>X1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>=</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>184</red>
        <green>201</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>184</red>
        <green>201</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>184</red>
        <green>201</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>184</red>
        <green>201</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Inicia">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Guarda">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cargar Datos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X4:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ingrese los valores de cada ecuaci√≥n</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);</string>
    </property>
    <property name="text">
     <string>    Resultados</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>591</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>label_6</zorder>
   <zorder>label</zorder>
   <zorder>Limpia</zorder>
   <zorder>label_3</zorder>
   <zorder>show3</zorder>
   <zorder>Sale</zorder>
   <zorder>Tabla</zorder>
   <zorder>Inicia</zorder>
   <zorder>show2</zorder>
   <zorder>show1</zorder>
   <zorder>Guarda</zorder>
   <zorder>label_4</zorder>
   <zorder>label_2</zorder>
   <zorder>label_5</zorder>
   <zorder>show4</zorder>
   <zorder>label_7</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Sale</sender>
   <signal>clicked()</signal>
   <receiver>Gausss</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
